setup.py

```diff
@@ -61,17 +61,17 @@
     'Programming Language :: Python :: 2',
     'Programming Language :: Python :: 2.7',
     'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3.4',
     'Programming Language :: Python :: 3.5',
     'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3.7',
     'Programming Language :: Python :: Implementation :: CPython',
     'Topic :: Security',
     'Topic :: Software Development'
   ],
   packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests",
       "debian"]),
   install_requires=["six", "securesystemslib[crypto,pynacl]>=0.11.3", "attrs",
-                    "python-dateutil", "iso8601", "pathspec"],
+                    "python-dateutil", "iso8601", "pathspec", "subprocess32"],
```

Comment:
I guess we could install subprocess32 only in Python 2 (e.g. `"subprocess32;python_version<'3'"`), since we don't use it in Python 3. If we do, please also update `requirements.txt`.

Reply:
Please do this before merging